demo/redirect-flow-example/src/App.tsx

```diff
@@ -409,20 +423,44 @@ function App() {
       uiConsole("provider not initialized yet");
       return;
     }
-    const newChainConfig = {
-      chainNamespace: CHAIN_NAMESPACES.EIP155,
-      chainId: "0xCC", // hex of 1261120
-      rpcTarget: "https://opbnb-mainnet-rpc.bnbchain.org",
-      // Avoid using public rpcTarget in production.
-      // Use services like Infura, Quicknode etc
-      displayName: "opBNB Mainnet",
-      blockExplorer: "https://opbnbscan.com",
-      ticker: "BNB",
-      tickerName: "opBNB",
-    };
-    await coreKitInstance.switchChain(newChainConfig);
-    setProvider(coreKitInstance.provider);
-    uiConsole("Changed to Sepolia Network");
+    // const newChainConfig : CustomChainConfig = {
+    //   chainNamespace: CHAIN_NAMESPACES.EIP155,
+    //   chainId: "0xCC", // hex of 1261120
+    //   rpcTarget: "https://opbnb-mainnet-rpc.bnbchain.org",
+    //   // Avoid using public rpcTarget in production.
+    //   // Use services like Infura, Quicknode etc
+    //   displayName: "opBNB Mainnet",
+    //   blockExplorerUrl: "https://opbnbscan.com",
+    //   ticker: "BNB",
+    //   tickerName: "opBNB",
+    // };
+    // if (coreKitInstance.status === COREKIT_STATUS.LOGGED_IN) {
+    //   let localProvider = new EthereumSigningProvider({ config: { chainConfig: newChainConfig } });
+    //   localProvider.setupProvider(coreKitInstance);
+    //   setProvider(localProvider);
+    // }
+    // or
```

Comment:
Let's try to remove all commented out code before merging. Doesn't look good in production code...